sql/updates/world/master/2022_xx_xx_05_world.sql

```diff
@@ -0,0 +1,694 @@
+DELETE FROM `conditions` WHERE `SourceTypeOrReferenceId`=13 AND `SourceGroup`=1 AND `SourceEntry`=84111 AND `SourceId`=0 AND `ElseGroup`=0 AND `ConditionTypeOrReference`=31 AND `ConditionTarget`=0 AND `ConditionValue1`=3 AND `ConditionValue2`=44951 AND `ConditionValue3`=0;
+INSERT INTO `conditions` (`SourceTypeOrReferenceId`, `SourceGroup`, `SourceEntry`, `SourceId`, `ElseGroup`, `ConditionTypeOrReference`, `ConditionTarget`, `ConditionValue1`, `ConditionValue2`, `ConditionValue3`, `NegativeCondition`, `ErrorType`, `ErrorTextId`, `ScriptName`, `Comment`) VALUES 
+(13, 1, 84111, 0, 0, 31, 0, 3, 44951, 0, 0, 0, 0, '', 'Armoire Camera - Agatha');
+
+-- Reverse Cast Ride Vehicle
+DELETE FROM `spell_script_names` WHERE `spell_id` = 84109 AND `ScriptName` = 'spell_gen_reverse_cast_target_to_caster_triggered';
+INSERT INTO `spell_script_names` (`spell_id`, `ScriptName`) VALUES
+(84109, 'spell_gen_reverse_cast_target_to_caster_triggered');
+
+-- Despawn All Summons
+DELETE FROM `spell_script_names` WHERE `spell_id` = 84011 AND `ScriptName` = 'spell_gen_despawn_all_summons_owned_by_caster';
+INSERT INTO `spell_script_names` (`spell_id`, `ScriptName`) VALUES
+(84011, 'spell_gen_despawn_all_summons_owned_by_caster');
+
+DELETE FROM `conditions` WHERE `SourceTypeOrReferenceId`=13 AND `SourceGroup`=1 AND `SourceEntry`=84011 AND `SourceId`=0 AND `ElseGroup`=0 AND `ConditionTypeOrReference`=31 AND `ConditionTarget`=0 AND `ConditionValue1`=3 AND `ConditionValue2`=44951 AND `ConditionValue3`=0;
+INSERT INTO `conditions` (`SourceTypeOrReferenceId`, `SourceGroup`, `SourceEntry`, `SourceId`, `ElseGroup`, `ConditionTypeOrReference`, `ConditionTarget`, `ConditionValue1`, `ConditionValue2`, `ConditionValue3`, `NegativeCondition`, `ErrorType`, `ErrorTextId`, `ScriptName`, `Comment`) VALUES 
+(13, 1, 84011, 0, 0, 31, 0, 3, 44951, 0, 0, 0, 0, '', 'Despawn All Summons - Target Agatha');
+
+-- Hillsbrad Refugee (44954, 44966)
+UPDATE `creature_template` SET `AIName`='', `ScriptName`='npc_silverpine_hillsbrad_refugee' WHERE `entry` IN (44954, 44966);
+
+DELETE FROM `creature_equip_template` WHERE `CreatureID` = 44954;
+INSERT INTO `creature_equip_template` (`CreatureID`, `ID`, `ItemID1`, `ItemID2`, `ItemID3`, `VerifiedBuild`) VALUES 
+(44954, 1, 3346, 0, 0, 45745),
+(44954, 2, 1906, 0, 0, 45745),
+(44954, 3, 25587, 13604, 0, 45745),
+(44954, 4, 3367, 0, 0, 45745),
+(44954, 6, 2715, 0, 0, 45745),
+(44954, 7, 2704, 0, 0, 45745),
+(44954, 8, 2714, 0, 0, 45745);
+
+DELETE FROM `creature_equip_template` WHERE `CreatureID` = 44966;
+INSERT INTO `creature_equip_template` (`CreatureID`, `ID`, `ItemID1`, `ItemID2`, `ItemID3`, `VerifiedBuild`) VALUES 
+(44966, 1, 2714, 0, 0, 45745),
+(44966, 2, 3367, 0, 0, 45745),
+(44966, 3, 2704, 0, 0, 45745),
+(44966, 4, 3351, 0, 0, 45745),
+(44966, 6, 25587, 13604, 0, 45745),
+(44966, 7, 3346, 0, 0, 45745),
+(44966, 8, 1906, 0, 0, 45745);
+
+UPDATE `creature` SET `equipment_id`=FLOOR(RAND()*(7-1+1)+1) WHERE `id` IN (44954, 44966);
```

Comment:
interesting way for random equipment, but i'd just use `-1` as its randomly selecting equipment

Reply:
The reason I decided to use this formula is that equipment 8 is the lamp that the spawns that patrol use, which isn't used by any other, so this allows me to use a random between 1-7 😸

Reply:
This is why they don't have ID for equipment and all 3 items are linked directly to creature entry in Creature.db2, everything else is scripted

Reply:
Because only certain rows and columns are available in db2
Items are after entry
(44954, 1906, 0, 0, 0, 33978, 33979, 33980, 33981, 1, 1, 1, 1, "Hillsbrad Refugee", "", "", 0, 0),
(44966, 1906, 0, 0, 0, 33982, 33983, 33984, 33985, 1, 1, 1, 1, "Hillsbrad Refugee", "", "", 0, 0),

Reply:
Can you share a link to that data, please? I'm looking at Creature.db2 right now and I cannot find them.

Reply:
https://gist.github.com/Kinzcool/a8c9cf058bf8263a03b84463383d3a98